### ISSUE TITLE
Bug: classification.py throws TypeError

### ISSUE THREAD

**Issue by cAbhi15** [astrolabsoftware/fink-broker#267]
**Describe the issue**
The following example throws a TypeError
https://github.com/astrolabsoftware/fink-broker/blob/a3efb02e6ec3a0a67666b8379e9342947e54080f/fink_broker/classification.py#L350

**Error message**
```
TypeError: data argument can't be an iterator
```
**System Information:**
_Please update accordingly_
 - Operating System: [`Ubuntu 18.04.2`]:
 - Fink Version: [`0.1.0`]

**Comment by JulienPeloton** [comment]
Interesting @cAbhi15, the travis log did not show this before. What Python version and pandas version do you use?

**Comment by cAbhi15** [comment]
```
In [1]: import pandas as pd                                                     

In [2]: pd.__version__                                                          
Out[2]: '0.23.4'
In [3]: pd.DataFrame(zip([1, 2, 3], [4, 5, 6]))                                 
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-0228a2be48b1> in <module>
----> 1 pd.DataFrame(zip([1, 2, 3], [4, 5, 6]))

~/anaconda3/lib/python3.7/site-packages/pandas/core/frame.py in __init__(self, data, index, columns, dtype, copy)
    405                 mgr = self._init_dict({}, index, columns, dtype=dtype)
    406         elif isinstance(data, collections.Iterator):
--> 407             raise TypeError("data argument can't be an iterator")
    408         else:
    409             try:

TypeError: data argument can't be an iterator
```
Thanks @JulienPeloton , Yes, this is a problem caused by pandas version.

**Comment by cAbhi15** [comment]
The error arises for `pandas` version (<0.24). Still, it is safer to use `list(zip)` for backward compatibility.
Fixed in #268
